src/js/components/FrameworkConfigurationForm.js

```diff
@@ -198,16 +199,58 @@ export default class FrameworkConfigurationForm extends Component {
   }
 
   handleFormChange(form) {
-    const { formData, errorSchema } = form;
+    const { formData } = form;
 
-    const formErrors = {};
-    Object.keys(errorSchema).forEach(tab => {
-      formErrors[tab] = this.getTotalErrorsForLevel(errorSchema[tab]);
-    });
-
-    this.props.onFormErrorChange(formErrors);
     this.props.onFormDataChange(formData);
-    this.setState({ errorSchema });
+
+    if (this.props.liveValidate) {
+      // When liveValidate is true, errorSchema is available for each change
+      this.probeErrorsSchemaForm();
+    }
+  }
+
+  handleFormError() {
+    this.probeErrorsSchemaForm();
+  }
+
+  /* Unconventional code notes
```

Comment:
🎨 I would personally search this kind of message in the git commits rather then here as it might be prone to becoming stale

Reply:
OK, good to know. I will squash this out. It is already a commit message of 
db14226.